src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/MissingDiagnosticAnalyzerAttributeRuleTests.cs

```diff
@@ -120,15 +120,15 @@ Imports Microsoft.CodeAnalysis
 Imports Microsoft.CodeAnalysis.Diagnostics
 
 Class MyAnalyzer
-	Inherits DiagnosticAnalyzer
-	Public Overrides ReadOnly Property SupportedDiagnostics() As ImmutableArray(Of DiagnosticDescriptor)
-		Get
-			Throw New NotImplementedException()
-		End Get
-	End Property
+    Inherits DiagnosticAnalyzer
+    Public Overrides ReadOnly Property SupportedDiagnostics() As ImmutableArray(Of DiagnosticDescriptor)
+        Get
+            Throw New NotImplementedException()
+        End Get
+    End Property
 
-	Public Overrides Sub Initialize(context As AnalysisContext)
-	End Sub
+    Public Overrides Sub Initialize(context As AnalysisContext)
+    End Sub
```

Comment:
⚠️ All of this indentation should not have changed. The indentation is located within string literals. Applies to multiple files.